README.md

```diff
@@ -72,6 +72,14 @@ hip_mandrill:
         sender: info@example.com
         sender_name: John Doe
         subaccount: Project # Optionally define a subaccount to use
+    proxy:
+        use: true # when you are behing a proxy. Default value is 'false'
+        host: example.com
+        port: 80
+        user: john
+        password: doe123
+    ssl:
+        verify: true # verify the SSL certificats. Default value is 'true'
```

Comment:
In this case I would advise you to add the CA certificate of your proxy to your PHP application using  curl.cainfo in your php.ini. If your proxy uses a self-signed certificate this will work. If the certificate is expired your admin can probably install a new one.